### PR TITLE
CODE CLEANING: Plotting + PEP8

### PR DESCRIPTION
**Pull request by oualib**
- Correcting plot functions.
 - removing ugly code [elem for elem...]
 - handling correctly exceptions

### CODE REVIEW

File: verticapy/core/string_sql/base.py [hunk 1]
```diff
@@ -100,7 +100,7 @@ def _in(self, *args) -> "StringSQL":
         elif len(args) == 0:
             ValueError("Method 'in_' doesn't work with no parameters.")
         else:
-            x = [elem for elem in args]
+            x = copy.deepcopy(args)
```
Comment (mail4umar) [inline verticapy/core/string_sql/base.py:103]: in a previous place, you used list() to replace the [elem ...]

File: verticapy/core/vdataframe/_corr.py [hunk 2]
```diff
@@ -1367,8 +1369,8 @@ def acf(
             result = self._new_vdataframe(query).corr(
                 [], method=method, focus=column, show=False
             )
-            columns = [elem for elem in result.values["index"]]
-            acf = [elem for elem in result.values[column]]
+            columns = copy.deepcopy(result.values["index"])
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_corr.py:1372]: does this also gives a list?

File: verticapy/core/vdataframe/_fill.py [hunk 3]
```diff
@@ -453,7 +456,7 @@ def fillna(
                         sql_push_ext=self._parent._vars["sql_push_ext"],
                         symbol=self._parent._vars["symbol"],
                     )
-                except:
+                except QueryError:
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_fill.py:459]: You are importing it as vQueryError

**Review by mail4umar — APPROVED** [review]

File: verticapy/core/vdataframe/_corr.py (hunk 2 above)
Comment (oualib) [reply verticapy/core/vdataframe/_corr.py:1372]: yes

File: verticapy/core/vdataframe/_fill.py (hunk 3 above)
Comment (oualib) [reply verticapy/core/vdataframe/_fill.py:459]: there are two different QueryError one from VerticaPy and one from Vertica-Python.

File: verticapy/core/string_sql/base.py (hunk 1 above)
Comment (oualib) [reply verticapy/core/string_sql/base.py:103]: it depends on the use-case. List will also convert dict to list.

**Pull request merged**